scripts/tn_bounce.sh

```diff
@@ -1,10 +1,47 @@
 #!/bin/bash
-pkill -9 eosd
-sleep 3
-now=`date +'%Y_%m_%d_%H_%M_%S'`
-DD=`ls -d tn_data_??`
-log=$DD/stderr.$now.txt
-nohup programs/eosd/eosd --resync --data-dir $DD > $DD/stdout.txt  2> $log &
-echo $! > $DD/eosd.pid
-rm $DD/stderr.txt
-ln -s stderr.$now.txt $DD/stderr.txt
+#
+# tn_bounce is used to restart a node that is acting badly or is down.
+# usage: tn_bounce.sh [arglist]
+# arglist will be passed to the node's command line. First with no modifiers
+# then with --replay and then a third time with --resync
+#
+# the data directory and log file are set by this script. Do not pass them on
+# the command line.
+#
+# in most cases, simply running ./tn_bounce.sh is sufficient.
+#
```

Comment:
I would note that this is called from launcher.

Reply:
Both of these scripts can also be called directly.